packages/actions/src/debug/debugTraceBlockProcedure.spec.ts

```diff
@@ -27,6 +27,40 @@ describe('debugTraceBlockJsonRpcProcedure', () => {
 		})
 		await mineHandler(client)({})
 
+		expect(
+			await procedure({
+				jsonrpc: '2.0',
+				method: 'debug_traceBlock',
+				params: [
+					{
+						blockTag: 'latest',
+					},
+				],
+				id: 1,
+			}),
+		).toMatchSnapshot()
+	})
+
+	it.todo('should trace transactions in a block and return the expected result', async () => {
```

Comment:
```suggestion
	it('should trace transactions in a block and return the expected result', async () => {
```

is this intentional?